reference.html

```diff
@@ -116,157 +116,157 @@ <h2 id="errors-and-exceptions"><a href="07-errors.html">Errors and Exceptions</a
 <h2 id="glossary">Glossary</h2>
```

Comment:
Not sure why `make preview` rendered this differently than the version in the repository, but I expect it's correct.